src/instructlab/lab.py

```diff
@@ -4,7 +4,6 @@
 import typing
 
 # Third Party
-from click_didyoumean import DYMGroup
```

Comment:
I think the `click_didyoumean` functionality is worth keeping.

I found a way to do aliasing with a much smaller change and without removing `DYMGroup` in #1264.

For reference for anyone else looking ... https://github.com/click-contrib/click-didyoumean

Reply:
dym is still in the resource groups which is where the cmds actually are.

I can add something like dym to my custom class :shrug:

Reply:
I think it's at least worth a follow-up issue to re-add the UX polish provided by didyoumean to the top-level

Reply:
https://github.com/instructlab/instructlab/issues/1325